subworkflows/local/fastq_align_bwamem_mem2_dragmap_sentieon/main.nf

```diff
@@ -23,13 +26,18 @@ workflow FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP {
     BWAMEM1_MEM(reads, index.map{ it -> [ [ id:'index' ], it ] }, sort) // If aligner is bwa-mem
     BWAMEM2_MEM(reads, index.map{ it -> [ [ id:'index' ], it ] }, sort) // If aligner is bwa-mem2
     DRAGMAP_ALIGN(reads, index.map{ it -> [ [ id:'index' ], it ] }, sort) // If aligner is dragmap
+    // The sentieon-bwamem-module does sorting as part of the conversion from sam to bam.
+    SENTIEON_BWAMEM(reads, index.map{ it -> [ [ id:'index' ], it ] }, fasta.map{fa -> [[:], fa]}, fasta_fai.map{fai -> [[:], fai]}) // If aligner is sentieon-bwamem
 
     // Get the bam files from the aligner
     // Only one aligner is run
     bam = Channel.empty()
     bam = bam.mix(BWAMEM1_MEM.out.bam)
     bam = bam.mix(BWAMEM2_MEM.out.bam)
     bam = bam.mix(DRAGMAP_ALIGN.out.bam)
+    bam = bam.mix(SENTIEON_BWAMEM.out.bam_and_bai.map{ meta, bam, bai -> [ meta, bam ] })
+
```

Comment:
```suggestion
 bai = Channel.empty()
```
If we need that "initialization" for for the `bam` channel, don't we also need it for the `bai` channel?

Reply:
I could swear we talked about this before :D . Mostly a matter of taste here. Don't really care either way